src/tests/ftest/process_core_files.py

```diff
@@ -258,9 +260,7 @@ def install_debuginfo_packages(self):
                 self.log, " ".join(["rpm", "-q", "--qf", "'%{evr}'", "daos"]), check=False)
             rpm_version = output.stdout
             cmds.append(
-                ["sudo", "dnf", "debuginfo-install", "-y"] + dnf_args
-                + ["daos-client-" + rpm_version, "daos-server-" + rpm_version,
-                   "daos-tests-" + rpm_version])
+                ["sudo", "dnf", "debuginfo-install", "-y"] + dnf_args + ["daos\*-" + rpm_version])
```

Comment:
Are you trying to use a wildcard to match daos-client-test and daos-server-test? If so, do you want "daos-\*" here?

Reply:
The wild card is being used to match multiple debuginfo packages, for example:
```
daos-admin-debuginfo-2.3.103-1.9099.gfacae803.el8.x86_64                      
daos-client-debuginfo-2.3.103-1.9099.gfacae803.el8.x86_64                     
daos-client-tests-debuginfo-2.3.103-1.9099.gfacae803.el8.x86_64               
daos-client-tests-openmpi-debuginfo-2.3.103-1.9099.gfacae803.el8.x86_64       
daos-debuginfo-2.3.103-1.9099.gfacae803.el8.x86_64                                                      
daos-firmware-debuginfo-2.3.103-1.9099.gfacae803.el8.x86_64                   
daos-serialize-debuginfo-2.3.103-1.9099.gfacae803.el8.x86_64                  
daos-server-debuginfo-2.3.103-1.9099.gfacae803.el8.x86_64                     
daos-server-tests-debuginfo-2.3.103-1.9099.gfacae803.el8.x86_64
```
which is essentially `daos{,-{admin,firmware,serialize,client{,-{tests,tests-openmpi}},server{,-tests}}}-debuginfo-2.3.103-1.9099.gfacae803.el8.x86_64` or just `daos*-2.3.103-1.9099.gfacae803.el8.x86_64`

Reply:
Ah, we wouldn't want to match something like `daosfoo-2.3.103-1.9099.gfacae803.el8.x86_64` (if that ever existed).  Thanks for the comment.